static/app/views/releases/drawer/releasesDrawerTable.tsx

```diff
@@ -53,30 +53,46 @@ const BASE_COLUMNS: Array<GridColumnOrder<keyof ReleaseHealthItem>> = [
  * can't re-use because this will eventually be a bit different,
  * especially with the in-drawer navigation.
  */
-export function ReleaseDrawerTable({start, end}: Props) {
+export function ReleaseDrawerTable({start, onSelectRelease, end}: Props) {
   const location = useLocation();
   const navigate = useNavigate();
-  const {releaseData, isLoading, isError, pageLinks} = useOrganizationReleases({
-    filters: [],
-    dateRange: {
-      start,
-      end,
-      statsPeriod: undefined,
-    },
-  });
+  const organization = useOrganization();
+  const {data, isLoading, isError /* isPending, getResponseHeader */} = useApiQuery<
+    Release[]
+  >(
+    [
+      `/organizations/${organization.slug}/releases/`,
+      {
+        query: {
+          ...Object.fromEntries(
+            Object.entries(location.query).filter(([key]) =>
+              ['project', 'environment'].includes(key)
+            )
+          ),
```

Comment:
start/end is going to come from the bubble that was clicked (soon to be from URL params). for most of these components we will need separate URL params (e.g. pagination we don't want to collide with main page)